Improve documentation on cache ttl (#1801)
nestjs/docs.nestjs.com: content/techniques/caching.md

await this.cacheManager.set('key', 'value');
```

The default expiration time of the cache is 5 seconds.

You can manually specify a TTL (expiration time) for this specific key, as follows:

```typescript
await this.cacheManager.set('key', 'value', { ttl: 1000 });
```

> You can set the expiration time to be effectively infinite as follows:

Reviewer: ```suggestion
You can disable the cache by setting the ttl to `null`:
```

Reviewer 2: These are two contrary sentences^ (infinite vs disabled)

Reviewer: damn, you are right.. i meant it the other way, haha..

Reviewer: corrected my suggestion

> You can disable expiration of the cache by setting the `ttl` to `null`:

Reviewer 2: ```suggestion
To disable expiration of the cache, set the `ttl` configuration property to `null`:
```
